packages/reporters/dev-server/src/Server.js

```diff
@@ -176,6 +185,21 @@ export default class Server extends EventEmitter {
     );
   }
 
+  getFilePathFromBundleGraph(name: string) {
```

Comment:
Possibly not needed now that we're using the distDir from options?

Reply:
I removed it.